crates/torii/core/src/processors/erc20_legacy_transfer.rs

```diff
@@ -40,18 +40,27 @@ where
         db: &mut Sql,
         _block_number: u64,
         block_timestamp: u64,
-        _event_id: &str,
+        event_id: &str,
         event: &Event,
     ) -> Result<(), Error> {
         let token_address = event.from_address;
         let from = event.data[0];
         let to = event.data[1];
+        let transaction_hash = event_id.split(':').nth(1).unwrap();
```

Comment:
Same here, the `id` construction could be abstracted and would make it easier to modify in the future. 👍